packages/framework/build.xml

```diff
@@ -78,7 +78,7 @@
         <echo level="warning" message="This phing target is deprecated since Shopsys Framework 7.3.0, use 'build-demo-ci' instead."/>
     </target>
 
-    <target name="build-demo-dev" depends="build-version-generate,wipe-excluding-logs,composer-dev,npm,timezones-check,dirs-create,test-dirs-create,assets,db-demo,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,tests-acceptance-build,clean-redis-old,checks-diff" description="Builds application for development with clean demo DB and runs checks on changed files."/>
+    <target name="build-demo-dev" depends="build-version-generate,wipe-excluding-logs,composer-dev,npm,timezones-check,dirs-create,test-dirs-create,assets,db-demo,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,tests-acceptance-build,clean-redis-old,checks" description="Builds application for development with clean demo DB and runs checks on changed files."/>
```

Comment:
Hm - the build will take more time now, I do like that :disappointed: 

I thought the Windows/Mac problem is covered by the info messages added in https://github.com/shopsys/shopsys/commit/fc50b78c9832a61faad8679ed5b3d1215e2ca51d

However, if this is the way we want to go (if I am outvoted by the rest of the team), you need to fix the description so it is not talking about the changed files anymore :wink:

Reply:
Well, imagine a situation you want to try Framework on Mac, you don't know anything about it and you run the installation script and the outcome of this is "error do something". It doesn't feel right :(

I didn't know we (the team) already had this discussion. I'm thinking about some conditions or something like that...

But currently the installation simply does not work on Mac.

Reply:
Maybe, we should use `build-demo-dev-quick` in installation guides, because probably people do not want run tests when they first install that.

Reply:
Running `build-demo-dev-quick` and `error-pages-generate` in the installation script and mention this in the installation guide seems like the okayest solution right now.
What do you say, @vitek-rostislav?

Reply:
oh daaamn we forgot about this https://github.com/shopsys/shopsys/commit/fc50b78c9832a61faad8679ed5b3d1215e2ca51d#diff-3c2478376ba4cacfb00c91ceb062a5a2
this should be also mentined in upgrade

Reply:
As the build.xml file is now in framework, I don't think the upgrade notes are necessary.

Reply:
I like it :slightly_smiling_face: